Simulator/Core/Core.cpp

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
Let's change this file name to Graphitti_Main.cpp

Reply:
Modified the filename from` Core.cpp` to `Graphitti_main.cpp`.